src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
```

Comment:
Regarding PSR 1 / 4 / other coding standards / conventions – is it decided if a space should be after declare? Not sure on that.